docs/source/daml/reference/contract-keys.rst

```diff
@@ -50,12 +50,11 @@ Use ``fetchByKey`` to fetch the ID and data of the contract with the specified k
 
 It returns a tuple of the ID and the contract object (containing all its data).
 
-You need authorization from **at least one** of the maintainers to run ``fetchByKey``. A maintainer can authorize by being a signatory, or by submitting the command/being a controller for the choice.
+``fetchByKey`` is authorized like ``fetch`` so you need to be a stakeholder to run ``fetchByKey``.
```

Comment:
I'm not sure this conveys the correct meaning. In particular, it's not clear who is "you". The authorization context might contain multiple parties. In my understanding, the that context needs to contain at least one stakeholder for a `fetchByKey` to be well-authorized. There are no conditions on the submitter though. 
@bame-da can you confirm?